python/tskit/trees.py

```diff
@@ -952,6 +952,17 @@ def time(self, u):
         """
         return self._ll_tree.get_time(u)
 
+    def depth(self, u):
+        """
+        Returns the number of nodes above the specified node on
```

Comment:
The word "node" is a bit overused here, how about "Returns the number of nodes on the path from ``u`` to root.`` This seems quite clear and concise.